components/dash-core-components/tests/integration/dropdown/test_dropdown_radioitems_checklist_option_title_prop.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: UTF-8 -*-
```

Comment:
The filename is a bit long and there is now more than one component tested, maybe move up one directory and rename to `test_title_props.py` ?